sdk/data/azcosmos/cosmos_client_options.go

```diff
@@ -15,4 +15,6 @@ type ClientOptions struct {
 	EnableContentResponseOnWrite bool
 	// PreferredRegions is a list of regions to be used when initializing the client in case the default region fails.
 	PreferredRegions []string
+	// When DisableCrossRegionalRetries is true the client will not apply the client retry policy, and will fail requests that would otherwise utilize cross-regional retries to attempt successful status.
+	DisableCrossRegionalRetries bool
```

Comment:
No public API changes

Reply:
are we enabling this by default for everyone then?

Reply:
The PR is enabling this by default, right? What I'm saying is that we can defer the public API name and public API review to another PR.